MAUI/Cartesian-Charts/Annotation.md

```diff
@@ -0,0 +1,743 @@
+
+* [RectangleAnnotation]() - Used to draw a rectangle over the chart area.
+* [EllipseAnnotation]() - Used to draw a circle or an ellipse over the chart area.
+* [LineAnnotation]() - Used to draw a line over the chart area.
+* [VerticalLineAnnotation]() - Used to draw a vertical line across the chart area.
+* [HorizontalLineAnnotation]() - Used to draw a horizontal line across the chart area.
+
+The following APIs are commonly used in all ShapeAnnotation:
+
+* [X2]() - Represents the X2 coordinate of the shape annotation.
+* [Y2]() - Represents the Y2 coordinate of the shape annotation.
+* [Fill]() - Represents the background color of the shape annotation.
+* [Stroke]() - Represents the stroke color of the shape annotation.
+* [StrokeWidth]() - Represents the stroke width of the shape annotation.
+* [StrokeDashArray]() - Represents the stroke dash pattern of the shape annotation.
+* [Text]() - Represents the annotation text of the shape annotation.
+* [LabelStyle]() - Represents the style for customizing the annotation text of the shape annotation.
+
+## Rectangle annotation
+
+The [RectangleAnnotation]() is used to draw a rectangle or a square at specific points in the chart area.
+
+{% tabs %}
+
+ {% highlight xaml %}
+
+        <chart:SfCartesianChart>
+        ...
+
+    <chart:SfCartesianChart.Annotations>
+        <chart:RectangleAnnotation X1="1" Y1="40" X2="2" Y2="20"/>
+    </chart:SfCartesianChart.Annotations>
+
+    </chart:SfCartesianChart>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+    SfCartesianChart chart = new SfCartesianChart();
+
+    var rectangle = new RectangleAnnotation()
+    {
+        X1 = 1,
+        Y1 = 40,
+        X2 = 2,
+        Y2 = 20,                
+    };
+
+    chart.Annotations.Add(rectangle);
+
+    this.Content = chart;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+## Ellipse annotation
+
+The [EllipseAnnotation]() is used to draw an oval or a circle at specific points in the chart area. You can also specify the height and width of the [EllipseAnnotation]() by using the Height and Width properties, respectively.
+
+{% tabs %}
+
+ {% highlight xaml %}
+
+        <chart:SfCartesianChart>
+        ...
+        <chart:SfCartesianChart.Annotations>
+            <chart:EllipseAnnotation X1="2" X2="4" Y1="10" Y2="15"/>
+        </chart:SfCartesianChart.Annotations>
+
+    </chart:SfCartesianChart>
+
+{% endhighlight %} 
+
+{% highlight c# %}
+
+    SfCartesianChart chart = new SfCartesianChart();
+
+    var ellipse = new EllipseAnnotation()
+    {
+        X1 = 2,
+        Y1 = 10,
+        X2 = 4,
+        Y2 = 15,    
+    };
+
+    chart.Annotations.Add(ellipse);
+
+    this.Content = chart;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Ellipse annotation in MAUI Chart](Annotation_image/default_ellipse.png)
+
+N> When the [X2]() and [Y2]() properties of the [EllipseAnnotation]() are set, the [Height]() and [Width]() properties do not work.
+
+## Line annotation
```

Comment:
Mention note for line stroke and Fill behavior

Reply:
Mentioned the note for fill for line ,horizontal and vertical